packages/terra-base/tests/nightwatch/Anchors.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Base from '../../lib/Base';
+
+const AnchorTests = () => (
+  <Base>
+    <br />
+    <a rel="noreferrer noopener" target="_blank" href="https://my.cerner.com">hyperlink</a>
```

Comment:
Let's update the href attribute to `https://my.cerner.com/?`.
Currently this link will display as a visited link if the user has been to https://my.cerner.com/.
Adding the `/?` helps as most people will not have gone to that URL unless the click on the hyperlink in this test page.

It is not as noticeable that this link and the visited link appear the same in this theme, though it is more noticeable in the consumer theme:

<img width="145" alt="screen shot 2018-01-29 at 1 43 30 pm" src="https://user-images.githubusercontent.com/633148/35530715-8081d792-04fa-11e8-858c-8c1aac9a350f.png">

Reply:
Updated https://github.com/cerner/terra-core/pull/1171/commits/a9401c240610fba74ffaa99ce4e0cd4f5f1040b0#diff-a1f3de17c0e4db0f893ccedc162dcd67R6